src/node.js

```diff
@@ -20,6 +20,40 @@
     });
     EventEmitter.call(process);
 
+    // Default process warning handler..
+    process.on('warn', (warning) => {
+      const cluster = NativeModule.require('cluster');
```

Comment:
Calling `NativeModule.require('cluster')` every time probably isn't great for performance.  What's the expected call frequency for this function?